pkg/acceptance/terrafarm/farmer.go

```diff
@@ -469,34 +469,63 @@ func (f *Farmer) Restart(ctx context.Context, i int) error {
 		return err
 	}
 
+	done := make(chan error, 1)
 	{
 		s, err := c.NewSession()
 		if err != nil {
 			return err
 		}
 		defer s.Close()
 
+		var outBuf, errBuf bytes.Buffer
+		s.Stdout = &outBuf
+		s.Stderr = &errBuf
+
 		if err := s.Start(cmd); err != nil {
 			return errors.Wrap(err, cmd)
 		}
+		go func() {
+			err := s.Wait()
```

Comment:
Because we don't want to evaluate `{out,err}Buf.String()` until after this returns.